website/docs/docs/build/custom-aliases.md

```diff
@@ -6,18 +6,22 @@ id: "custom-aliases"
 
 ## Overview
 
-When dbt runs a model, it will generally create a relation (either a `table` or a `view`) in the database. By default, dbt uses the filename of the model as the identifier for this relation in the database. This identifier can optionally be overridden using the [`alias`](/reference/resource-configs/alias) model configuration.
+When dbt runs a model, it will generally create a relation (either a <Term id="table" /> or a <Term id="view" />) in the database, or in the case of an [ephemeral model](/docs/build/materializations#ephemeral) it will create a <Term id="cte" /> for use in another model. By default, dbt uses the filename of the model as the identifier for the relation or CTE it creates. This identifier can optionally be overridden using the [`alias`](/reference/resource-configs/alias) model configuration.
```

Comment:
```suggestion
When dbt runs a model, it will generally create a relation (either a <Term id="table" /> or a <Term id="view" />) in the database, except in the case of an [ephemeral model](/docs/build/materializations#ephemeral), when it will create a <Term id="cte" /> for use in another model. By default, dbt uses the model's filename as the identifier for the relation or CTE it creates. This identifier can be overridden using the [`alias`](/reference/resource-configs/alias) model configuration.
```